components.nightly.yaml

```diff
@@ -17,10 +17,13 @@ chains:
   version: nightly
 results:
   github: tektoncd/results
-  version: v0.5.0
+  version: v0.10.0
```

Comment:
@khrm why `results` `manual-approval-gate`, and `hub` components has version number for nightly?

Reply:
@jkandasa because they do not "publish" nightly release yaml somewhere (hence my [comment above](https://github.com/tektoncd/operator/pull/2161#issuecomment-2122300440)).